packages/trip-form/src/types.ts

```diff
@@ -107,11 +107,14 @@ export interface ModeSelectorOptionSet {
   tertiary?: ModeSelectorOption[];
 }
 
-type SimpleStyledDiv = StyledComponent<"div", any>;
+type SimpleStyledDiv = StyledComponent<
+  "div",
+  any,
+  { baseColor?: string | undefined }
```

Comment:
Remove `undefined` (`undefined` is implied in the `?:` syntax).

```suggestion
  { baseColor?: string }
```